### PR TITLE
implement: tutorial (En) template

### PR DESCRIPTION
**Pull request by inwookie**
## Tutorial Template Implementation (EN) 
1. ml 
   - [x] speech recognition
   - [x] speech recognition2
   - [x] automl regression
   - [x] automl classification 
   - [x] image classification 
   - [x] text classification
   - [x] visual question answering
   - [ ] udm - HOLD 
2. search
    - [x] search_image_by_image
    - [x] search_image_by_keyword
    - [x] search_image_by_text
    - [x] search_text_by_text
    - [x] search_video_by_text

### CODE REVIEW

File: tutorial/thanosql_ml/classification/automl_classification.ipynb [hunk 1]
```diff
@@ -911,7 +911,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.10.8 64-bit (microsoft store)",
```
Comment (yeemh) [inline tutorial/thanosql_ml/classification/automl_classification.ipynb:914]: 합칠 때 conflict 날 수 있으니 롤백해주세요

**Review by yeemh — COMMENTED** [review]
나머지는 vscode로 확인해볼게요

File: tutorial/thanosql_ml/classification/automl_classification.ipynb (hunk 1 above)
Comment (inwookie) [reply tutorial/thanosql_ml/classification/automl_classification.ipynb]: revert 했습니다

**Review by yeemh — APPROVED** [review]
WE DID IT!

**Pull request merged**